conf/messages.en

```diff
@@ -179,7 +179,7 @@ view.new-or-existing-case.heading=What do you want to do?
 view.new-or-existing-case.option.new=Send documents for a new customs check
 view.new-or-existing-case.option.new.hint=This includes sending C1601, C1602 and C1603 forms, or cancelling a case.
 view.new-or-existing-case.option.existing=Add extra information to an existing customs check
-view.new-or-existing-case.option.existing.hint=If you already have a case reference number, you can add more information to the case.
+view.new-or-existing-case.option.existing.hint=Add more information to an already-submitted customs check.
```

Comment:
Sorry wrong content changed - the hint text itself won't be changing so this content just needs to replace the radio button content instead (ie the line above)

The full stop won't be needed on the radio text so we can also drop that